src/backend/distributed/sql/citus--8.3-1--9.0-1.sql

```diff
@@ -1,6 +1,9 @@
 /* citus--8.3-1--9.0-1 */
 
-/* bump version to 9.0-1 */
+-- We swapped the groupid and nodeid sequences when creating pg_dist_node
+ALTER TABLE pg_dist_node ALTER COLUMN groupid SET DEFAULT nextval ('pg_dist_groupid_seq');
+ALTER TABLE pg_dist_node ALTER COLUMN nodeid SET DEFAULT nextval('pg_dist_node_nodeid_seq');
```

Comment:
Do you know if this can cause problems for upgrades? If so, maybe also `ALTER SEQUENCE ... RENAME ...` or something so `groupid` and `nodeid` assignemnt continue gracefully?

Reply:
`master_add_node` already uses the right sequences (e.g. `pg_dist_node_nodeid_seq` for `nodeid`) so if we rename them it will start using a different sequence for the nodeid. The ALTER TABLE is there to fix direct inserts and for not looking weird.